src/EnergyPlus/EvaporativeFluidCoolers.cc

```diff
@@ -1954,7 +1954,7 @@ namespace EvaporativeFluidCoolers {
                 this->inletConds.AirPress = state.dataEnvrn->StdBaroPress;
                 this->inletConds.AirHumRat =
                     Psychrometrics::PsyWFnTdbTwbPb(state, this->inletConds.AirTemp, this->inletConds.AirWetBulb, this->inletConds.AirPress);
-                auto f = [&state, this, &DesEvapFluidCoolerLoad, &par1, &par2, &Cp](Real64 UA) {
+                auto f = [&state, this, DesEvapFluidCoolerLoad, par1, par2, Cp](Real64 UA) {
```

Comment:
Finished cleaning up the reference captures that I had already made in this branch.